dev/tools/gen_defaults/lib/template.dart

```diff
@@ -31,6 +34,15 @@ abstract class TokenTemplate {
   /// Defaults to 'Theme.of(context).textTheme.'
   final String textThemePrefix;
 
+  /// Check if a token is available.
+  bool tokenAvailable(String tokenName) => _tokens.containsKey(tokenName);
+
+  /// Resolve a token while logging its usage.
+  dynamic getToken(dynamic tokenName) {
```

Comment:
Should these two `dynamic` just be `String`?

Reply:
Yes for `tokenName`, no for the return type, which can be other types (e.g. `double`)